src/components/AI/Form/Form.tsx

```diff
@@ -135,27 +88,56 @@ const Form = ({ onSubmit }: IFormProps) => {
         <Typography variant="h3" sx={{ my: 2, textAlign: "center" }}>
           Register the AI
         </Typography>
-
-        <CustomFormControl
-          label={CHECK_AND_RADIO[0].label}
-          values={CHECK_AND_RADIO[0].value}
-          selectedValues={Ai.levelOfAI}
-          multiple={true}
-          onChange={onLevelChange}
+        {errors.levelOfAI && (
+          <FormHelperText>{errors.levelOfAI.message}</FormHelperText>
+        )}
+        <Controller
+          name="levelOfAI"
+          control={control}
+          rules={{ required: "This field is required" }}
+          render={({ field }) => (
+            <CustomFormControl
+              label={CHECK_AND_RADIO[0].label}
+              values={CHECK_AND_RADIO[0].value}
+              selectedValue={levelOfAI}
+              multiple={true}
+              onChange={(newOption) => field.onChange(newOption)}
+            />
+          )}
         />
-        <CustomFormControl
-          label={CHECK_AND_RADIO[1].label}
-          values={CHECK_AND_RADIO[1].value}
-          selectedValues={Ai.whereAIIsUsed}
-          multiple={true}
-          onChange={onWhereUsedChange}
+        {errors.whereAIIsUsed && (
+          <FormHelperText>{errors.whereAIIsUsed.message}</FormHelperText>
+        )}
```

Comment:
common practise in situations like this pass here blank string for preserve space for message like:

from
```
{/* {errors.levelOfAI && (
   <FormHelperText>{errors.levelOfAI.message}</FormHelperText>
)} */}
```
to 
```
<FormHelperText>{errors.levelOfAI?.message ?? " "}</FormHelperText>
```

also move helpertexts below input

https://github.com/proga-react-practice/xi-team-project/assets/103277184/6a193856-1f45-4a59-95d0-bbdd894ac48f